src/parser.js

```diff
@@ -0,0 +1,64 @@
+const compiler = require('vue-template-compiler')
+const LRU = require('lru-cache')
+const hash = require('hash-sum')
+const { SourceMapGenerator } = require('source-map')
+
+const cache = LRU(100)
```

Comment:
Why 100 max items in cache? Do you think memory will be a problem?

Reply:
I've kept same as vue-loader.